docs/readme.md

```diff
@@ -70,6 +70,7 @@ Release Notes
 ### Upcoming Release
 - Changes
     - Analytics: Added `SetConsent()` and `GetSessionIdAsync()` APIs.
+    - Crashlytics: Add `ReportUncaughtExceptionsAsFatal` propertie and `LogExceptionAsFatal` API. Crashlytics now support on-demand fatal reports.
```

Comment:
Hi @rachelsaunders, can you help to review this release notes? Apparently unity also automated their release notes generation.